akismet-privacy-policies.php

```diff
@@ -123,19 +184,21 @@ public function add_comment_notice( $arr_comment_defaults ) {
 			return $arr_comment_defaults;
 		}
 
-		$options = get_option( 'akismet_privacy_notice_settings' );
-		if ( ! isset( $options[ 'checkbox' ] ) || empty( $options[ 'checkbox' ] ) && 0 != $options[ 'checkbox' ] ) {
-			$options[ 'checkbox' ] = $this->checkbox;
+		// $locale = isset( $_GET[ 'translation' ]) ? $_GET[ 'lang'] : get_locale();
+		// $this->options = get_option( 'akismet_privacy_notice_settings_' . $this->translation );
+
+		if ( ! isset( $this->options[ 'checkbox' ] ) || empty( $this->options[ 'checkbox' ] ) && 0 != $this->options[ 'checkbox' ] ) {
+			$this->options[ 'checkbox' ] = $this->checkbox;
 		}
-		if ( empty( $options[ 'notice' ] ) ) {
-			$options[ 'notice' ] = $this->notice;
+		if ( empty( $this->options[ 'notice' ] ) ) {
+			$this->options[ 'notice' ] = __( $this->notice );
```

Comment:
Need escaping.

Reply:
Done

> In this context of PR we should also add a new wiki entry for the language EN - https://github.com/inpsyde/Akismet-Privacy-Policies/wiki

I'll do that as soon as I find time.